extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -199,7 +238,9 @@ const sopClassUids = [
   sopClassDictionary.EnhancedUSVolumeStorage,
 ];
 
-function getSopClassHandlerModule() {
+function getSopClassHandlerModule(appContextParam) {
+  appContext = appContextParam;
```

Comment:
I prefer prop drilling in this case: passing the extension manager below into the getDisplaySetsFromSeries, and then into makeDisplaySets and then the function you like